### ISSUE TITLE
Devise variant is consistently failing lighthouse accessibility audit

### ISSUE THREAD

**Issue by G-Rath** [ackama/rails-template#550]
It started happening a couple of weeks ago - I can't debug it as I can't run the lighthouse matchers locally due to a WSL config thing I've just not gotten around to digging into (standard js specs work fine, but however lighthouse launches the browser currently doesn't work on WSL)

**Comment by eoinkelly** [comment]
These specs pass locally for me. I'll continue investigating.

**Comment by eoinkelly** [comment]
Closed by #561
